tests/lib/rules/jsx-handler-names.js

```diff
@@ -101,6 +101,17 @@ ruleTester.run('jsx-handler-names', rule, {
     options: [{
       eventHandlerPropPrefix: false
     }]
+  }, {
+    code: '<TestComponent onClick={() => handleClick} />',
+    options: [{
+      checkInlineFunction: true
+    }]
+  }, {
+    // TODO: need to fix
+    code: '<TestComponent onClick={() => handleClick} />',
```

Comment:
테스트코드에서 질문은 checkInlineFunction 옵션이 false일 경우 valid하게 해야하는데 
```suggestion
    code: '<TestComponent onClick={handleClick} />',
```
이런식으로 수정하는게 맞을까요? 그리고 현재 rule에서 onClick을 사용하는 테스트 코드가 없던데 onClick을 사용해도 될까요?

그리고 36번째 줄 41번째 줄에 보면 checkLocalVariables에 대한 테스트 코드를 볼 수 있는데 사실 checkLocalVariables이 무슨 역할을 하는 옵션인지 명확하게 이해하지 못하겠습니다.

Reply:
https://github.com/dididy/eslint-plugin-react/blob/bf9d01c88a0e13037454745717013c6d2ab5a2a4/tests/lib/rules/jsx-handler-names.js#L105
실수가 있어서 아래의 형태로 수정하였습니다.
`<TestComponent onClick={() => handleClick()} />`